code/modules/projectiles/guns/energy.dm

```diff
@@ -110,6 +112,9 @@
 /obj/item/gun/energy/load_ammo(var/obj/item/A, mob/user)
 	if(!istype(A, /obj/item/cell))
 		return
+	if(!istype(A, ammoType))
+		to_chat(user, "<span class='warning'>[src] can't use that power cell.</span>")//already a power cell here
```

Comment:
This var is redundant, instead of checking for a snowflake ammo type, just re-use the cell type. I cannot think of any instance where they would not be the same thing.